marshmallow_oneofschema/one_of_schema.py

```diff
@@ -159,6 +162,9 @@ def _load(self, data, partial=None, unknown=None):
         if self.type_field in data and self.type_field_remove:
             data.pop(self.type_field)
 
+        if self.nest_result and data['value'] is not None:
```

Comment:
Is this a potential `KeyError` if `'value'` isn't in data?
Should this be `data.get('value')` instead?

Reply:
+1, this way it properly raises a `ValidationError`